ckanext/data_qld/helpers.py

```diff
@@ -188,3 +188,7 @@ def is_delete_request():
 
 def is_api_request():
     return get_request_action() == 'action' or '/action/' in get_request_path()
+
+
+def get_old_data(data, context):
+    return toolkit.get_action('package_show')(context, {"id": data.get('id')})
```

Comment:
You can get the package object from the `context.get('package')` and fall back to `package_show` if for some reason it is not there